src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -454,11 +454,16 @@ private function buildTableRows($rows)
      * @param int   $line
      *
      * @return array
+     *
+     * @throws \UnexpectedValueException
      */
     private function fillNextRows(array $rows, $line)
     {
         $unmergedRows = array();
         foreach ($rows[$line] as $column => $cell) {
+            if (is_array($cell)) {
+                throw new \UnexpectedValueException('The cell should always be a string, array given.');
```

Comment:
be a string or a TableCell

Reply:
`if (!is_string($cell) && !$cell instanceof TableCell)` ?